src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
do not set the margin and padding to the `*` selector. Reset it for the list of tags you need (for example `body, p`)